docs/api_equilibrium.rst

```diff
@@ -53,12 +53,15 @@ profiles together by addition, multiplication, or scaling.
     :recursive:
     :template: class.rst
 
-    desc.profiles.PowerSeriesProfile
-    desc.profiles.SplineProfile
+    desc.profiles.HermiteSplineProfile
```

Comment:
I'd prefer to keep the old order here, which was meant to be ordered by importance/how often they get used. The one in `api.rst` is a full alphabetical list.